config.yml

```diff
@@ -671,6 +671,8 @@ nodes:
         type: location?
       - name: message_loc
         type: location?
+      - name: name
+        type: constant
```

Comment:
It seems when there is both a `constant` and `location` field the `location` is second, so I'll swap these, to be consistent with many other nodes which have it in that order.